smartcontract/contract/interactions/playground.sh

```diff
@@ -41,7 +41,7 @@ function convert_to_hex() {
 }
 
 function convert_contract_address_to_hex() {
-	echo "0x"$(erdpy wallet bech32 --decode $1)
```

Comment:
Please place the name of the executable in a variable to be able to quickly change it in all places